scripts/update-current-status.py

```
@@ -245,14 +263,48 @@ def _replace_block(text: str, begin_marker: str, end_marker: str, new_content: s
+    )
+    if cov.ux_percent >= lsp_target_pct:
+        lsp_target = "**Target**: maintain 100% LSP coverage (no regressions)"
+    else:
+        lsp_target = f"**Target**: 100% LSP coverage (from current {cov.ux_percent}%)"
 
+    bullets_content = "\n".join([
+        lsp_coverage_bullet,
+        protocol_compliance_bullet,
+        "",
+        lsp_target,
+    ])
```

Ruff (0.15.6)

[warning] Consider f-string instead of string join

Replace with f-string

(FLY002)